sql/synth_recipes.sql

```diff
@@ -2278,7 +2278,7 @@ INSERT INTO `synth_recipes` VALUES (31534,0,0,0,0,0,39,0,0,0,0,4099,4241,826,932
 INSERT INTO `synth_recipes` VALUES (31535,0,0,0,0,0,40,0,0,0,0,4099,4241,8854,0,0,0,0,0,0,0,2400,2400,2400,2400,3,3,3,3,'Shadow Roll');
 INSERT INTO `synth_recipes` VALUES (31536,0,0,0,0,0,40,0,0,0,0,4099,4241,820,820,827,850,0,0,0,0,2400,2400,2400,2400,3,3,3,3,'Shadow Roll');
 -- INSERT INTO `synth_recipes` VALUES (31537,0,0,0,0,0,40,0,0,255,0,4099,4241,825,826,1882,2507,4154,0,0,0,19252,19252,19252,19252,33,66,99,99,'Pet Poultice');
--- INSERT INTO `synth_recipes` VALUES (31538,0,0,0,0,255,40,0,0,0,0,4099,4241,661,819,826,827,827,0,0,0,12474,12479,12479,12479,1,1,1,1,'Wool Hat');
+INSERT INTO `synth_recipes` VALUES (31538,0,0,0,0,255,40,0,0,0,0,4099,4241,661,819,826,827,827,0,0,0,12474,12479,12479,12479,1,1,1,1,'Wool Hat');
```

Comment:
This is listing the requisite Goldsmithing as 255. It should be 8.

Reply:
ooof huge wiff on my part - thanks! updated.